rir/R/rir.R

```diff
@@ -43,3 +43,8 @@ rir.eval <- function(what, env = globalenv()) {
 rir.body <- function(f) {
     .Call("rir_body", f);
 }
+
+# breakpoint during evaluation
+# insert a call to `.debug.break()` in R code and get a breakpoint when the function is evaluated
+# this is an empty function because it gets directly lowered to the "int3_" bytecode
+.debug.break <- function() { }
```

Comment:
maybe put an `error("missed breakpoint")` here, so it's less surprising, if one forgets to compile the code?

Reply:
Is it to cumbersome to create a SEXP for an empty function? I'd remove the function from here and just compile a programatically created empty function with the proper symbol when the compiler initializes.

Reply:
I prefer @o-'s suggestion. It's simpler and less surprising (tracking down a programmatically-generated function is not very fun).